Sourcery/Parsing/FileParser.swift

```diff
@@ -234,8 +234,25 @@ extension FileParser {
 
     internal func parseVariable(_ source: [String: SourceKitRepresentable], isStatic: Bool = false) -> Variable? {
         guard let (name, _, accesibility) = parseTypeRequirements(source),
-            accesibility != .private && accesibility != .fileprivate,
-            let type = source[SwiftDocKey.typeName.rawValue] as? String else { return nil }
+            accesibility != .private && accesibility != .fileprivate else { return nil }
+
+        var maybeType: String? = source[SwiftDocKey.typeName.rawValue] as? String
```

Comment:
I think that might be good future improvement but it would require bigger changes as we'd need to correlate source with specific Structure.substructure, maybe think of that when we have another scenario where we could leverage Structure?